node/bin/src/config.rs

```diff
@@ -178,13 +178,16 @@ pub struct SequencerConfig {
     pub fee_collector_address: Address,
 
     /// Override for base fee (in wei). If set, base fee will be constant and equal to this value.
-    pub base_fee_override: Option<u128>,
+    #[config(default_t = None, with = Optional(Serde![str]))]
+    pub base_fee_override: Option<U256>,
```

Comment:
let's use U128, alloy types expect fees to be in u128 range so large U256 shouldn't be allowed